source/iostreams.tex

```diff
@@ -4600,8 +4600,8 @@
 To decide if the character \tcode{c} is a whitespace character,
```

Comment:
While we are here, we can also drop the quote of the constructor signature in the previous paragraph.

Like...it's a Remarks for this constructor. Do we really need to say yet again that we are talking about this constructor?